CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [1.11.7](https://github.com/iamkun/dayjs/compare/v1.11.6...v1.11.7) (2022-12-06)
```

Comment:
I think the changes to CHANGELOG.md should be reverted as they're not in scope for this PR

Reply:
This line comes from dayjs 1.11.7 - the base of my pr and is autogenerated by the dayjs build system (I suppose). The line itself is the link to the git changes (as in any versions before). But you knew all this before 😄 

So I don't know why this is a "changed file" here, as I didn't change it in my pr.
in the cuurent master branch (1.11.9) the line looks unchanged and AFAIK github should / could rebase a pr automatically on merge.
So what should I change in the changelog.md file?

Reply:
GitHub says these lines are added by your PR. Maybe if you rebase this PR to `dev` these changes would be removed since they are then already implemented

Reply:
Sounds good, as this pr will merge to dev.
But.... somehow it looks like dev is not up to date, For me it looks like the dev branch somehow is stuck on v1.11.6, while master moved on to 1.11.9.

So I think / hope that all this mess will disappear, when this pr is merged into master (if this will ever happen :-)

Reply:
Did you check out the `dev` from your fork by any chance? It seems that `dev` is the default branch of this repo and things have been merged to it a few days ago

Reply:
oh man this is a mess with the branches in this repo:
master contains the line to v1.11.9 while dev is not up to date with master.

I think I will rebase the pr to dev as soon as @iamkun answers to my comment on his remark to 'utils.js' below.